src/nativeresources/resourcestring.h

```diff
@@ -26,7 +26,7 @@ int LoadNativeStringResource(const NativeStringResourceTable &nativeStringResour
 #define NATIVE_STRING_RESOURCE_TABLE(name) CONCAT(nativeStringResourceTable_, name)
 
 #define DECLARE_NATIVE_STRING_RESOURCE_TABLE(name) \
-    extern const NativeStringResourceTable NATIVE_STRING_RESOURCE_TABLE(name)
+    __attribute__((visibility("default"))) extern const NativeStringResourceTable NATIVE_STRING_RESOURCE_TABLE(name)
```

Comment:
Still has the error message:
```
Undefined symbols for architecture x86_64:
  "_nativeStringResourceTable_mscorrc_debug", referenced from:
```
@jkotas @janvorli @AndyAyersMS  Any suggestion?

Reply:
@janvorli has the best context to comment on what may be going on here. He is on the plane right now. He should be able to respond tomorrow.

Reply:
@jkotas Thank you!

Reply:
@fiigii it looks like the attribute should be placed at the end of the declaration for variables. So
```C++
extern const NativeStringResourceTable NATIVE_STRING_RESOURCE_TABLE(name) __attribute__((visibility("default")))
```
I didn't have a chance to try it yet though.

Reply:
@janvorli Yes, you are right https://gcc.gnu.org/onlinedocs/gcc-3.2/gcc/Variable-Attributes.html 
Thank you so much!